aide_design/constants.py

```diff
@@ -6,6 +6,9 @@
 
 Last modified: Fri Jul 7 2017
 by: Sage Weber-Shirk
+
+This file contains constants which represent physical properties and scientific
```

Comment:
Make it [numpy style](https://github.com/numpy/numpy/blob/master/doc/example.py)